README.md

```diff
@@ -226,6 +226,51 @@ and React (using TSX files). Please use the React Generators for creation.
 * Push to the remote for `PRIVATE_PROD_DEPLOY` (ask Eric for the remote and access)
 * Push to heroku production  using `git commit production PRIVATE_PROD_DEPLOY:master` ( ask Eric for access to `production`)
 
+## (Mac Setup) Build for Production 
+# In order to get prod env set, you need to download Github CLI and Heroku CLI. 
+
+# Github CLI setup 
+* gh # to check if GH CLI has been downloaded 
+*  brew install gh  # if GH CLI has not been downloaded
+* gh 
+* gh auth
+* gh auth login. 
```

Comment:
Please wrap these lines in a multiline code block. You can do this by using three tickmarks (`) at the beginning of an otherwise empty line before the first shell command and then three tickmarks on the line after your shell commands

Also, please remove the leading *'s